client/src/components/RatingsAndReviews/RatingsAndReviews.jsx

```diff
@@ -7,13 +7,13 @@ const RatingsAndReviews = () => {
 
   const getReviews = (username) => {
     let options = {
-      url: 'https://app-hrsei-api.herokuapp.com/api/fec2/:rfp2305/',
+      url: 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/',
       headers: {
         'User-Agent': 'request',
-        'Authorization': `token ${process.env.TOKEN}`
+        'Authorization': `token ${process.env.TIMTOKEN}`
```

Comment:
Should leave it as "TOKEN" since we'll all have our respective .env files and if we each call the our github token "TOKEN" then the whole app will work. (Otherwise I'd have to create a variable called TIMTOKEN for this to work on my computer)

Reply:
Ah ok, I forgot that that is on the gitignore list.